sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -27,17 +27,35 @@ point of entry for receiving of any type (from single partition, all partitions,
 | `EventHubClient.createFromAadTokenCredentials()` | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [usingAadAuth](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/usingAadAuth.ts)
 | `EventProcessorHost.createFromConnectionString()`                           | `new EventHubConsumerClient(..., checkpointStore)`               | [receiveEventsUsingCheckpointStore](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEventsUsingCheckpointStore.ts) |
 
+Other noteworthy changes:
+- In v5, the `EventHubConsumerClient` class takes the consumer group name as a mandatory argument in its constructor.
+If you havent created any consumer groups explicitly, then use the name of the default consumer group which is `$Default`.
+- For checkpoint store implementation using Azure Storage Blobs, use the 
+[@azure/eventhubs-checkpointstore-blob](https://www.npmjs.com/package/@azure/eventhubs-checkpointstore-blob) package.
+
 ### Receiving events 
 
 | In v2                                          | Equivalent in v5                                                 | Sample |
 |------------------------------------------------|------------------------------------------------------------------|--------|
 | `EventHubClient.receive()` and `EventHubClient.receiveBatch()`                       | `EventHubConsumerClient.subscribe()`                               | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEvents.ts) |
 
+Other noteworthy changes:
+- Use the `options` parameter to the `subscribe()` method to specify starting position to receive events from.
+- The `subscribe()` method allows you to receive events in batches whose size can be configured using the `options` parameter.
+- The user provided function to process events will be invoked only after the previous invocation completes.
+This is different from v2 where the function was invoked for each event without waiting for the previous call to complete.
```

Comment:
Was it for each event or for each batch of events?

Reply:
Each event. The push model served one event at a time in v2